client/components/Auth/InputField.vue

```diff
@@ -1,10 +1,16 @@
 <template>
   <div>
-    <label class="font-semibold">{{ fieldName }}</label>
-    <ValidationProvider v-slot="{ errors, touched }" :vid="id" :rules="rules">
+    <label class="font-semibold" :for="fieldName">{{ fieldName }}</label>
+    <ValidationProvider
+      v-slot="{ errors, touched }"
+      slim
+      :vid="id"
+      :rules="rules"
+    >
       <div class="relative flex items-center -mt-2">
         <input
           v-if="fieldType != 'dropdown'"
```

Comment:
!== ?

Reply:
Good catch, I'll fix it.
This is from main btw.